tools/docker/iroha-all-in-one/healthcheck.py

```diff
@@ -0,0 +1,37 @@
+import os
+import sys
+from iroha import Iroha, IrohaCrypto, IrohaGrpc
+
+#setup of iroha client address
+IROHA_HOST_ADDR = os.getenv('IROHA_HOST_ADDR', '127.0.0.1')
+IROHA_PORT = os.getenv('IROHA_PORT', sys.argv[2]) 
+net = IrohaGrpc(f'{IROHA_HOST_ADDR}:{IROHA_PORT}')
+#setup admin account
+ADMIN_ACCOUNT_ID = os.getenv('ADMIN_ACCOUNT_ID', 'admin@test')
+ADMIN_PRIVATE_KEY = os.getenv(
+    'ADMIN_PRIVATE_KEY', 'f101537e319568c765b2cc89698325604991dca57b9716b58016b253506cab70')
+iroha_admin = Iroha(ADMIN_ACCOUNT_ID)
+user_account = 'admin@test'
+iroha_user = Iroha(user_account)
+
+#health check function
```

Comment:
I suggest to use docstring inside the function instead of the comment

Reply:
Resolved. Thank you!